tech/tech/api/orders_router.py

```
@@ -0,0 +1,151 @@
+        limit (int): Maximum number of orders to retrieve. Defaults to 10.
+        skip (int): Number of orders to skip before starting retrieval. Defaults to 0.
+        use_case (ListOrdersUseCase): Use case to handle the business logic for listing orders.
+
+    Returns:
+        OrderList: Paginated list of orders.
+    """
+    return {"orders": use_case.execute(limit, skip)}
+
+
+@router.put("/{order_id}", status_code=200)
+def update_order_status(order_id: int, status: OrderStatusEnum, use_case: UpdateOrderStatusUseCase = Depends(get_update_order_status_use_case)):
```

Ruff (0.8.2)

Line too long (144 > 79)

(E501)